scripts/api_wrapper.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+A wrapper script to interface with our manager app via API calls.
+The manager calls this script and passes the request as cmd line arguments.
+This wrapper starts the script described in the command line argument,
+and when that finishes, it makes an API call back to the manager.
+"""
+
+import logging
+import sys
+import requests
+import subprocess
+
+
+def main():
+    step_id = sys.argv[2]

```

Comment:
`argparse`

Reply:
When I first wrote this code it seemed easier to just use sys.argv, since most of the args will be passed down to the next script and there is an unknown number of them.

Then things got a bit more complicated. I'm still not sure is it worth using argparse...